rgw/admin/user_bucket.go

```diff
@@ -0,0 +1,71 @@
+//go:build ceph_preview
+// +build ceph_preview
+
+package admin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// BucketListingSpec describes a request
+type BucketListingSpec struct {
+	UID          string `url:"uid"`
+	GenerateStat *bool  `url:"stats"`
+}
+
+// ListUsersBuckets will return the list of all users buckets without stat
+// PREVIEW
+func (api *API) ListUsersBuckets(ctx context.Context, uid string) ([]string, error) {
+	if uid == "" {
+		return nil, errMissingUserID
+	}
+
+	generateStat := false
+	listingSpec := BucketListingSpec{
```

Comment:
IMO `listingSpec` is not required for this API, just pass the uid directly to request

Reply:
and what about `stats` query parameter?

Reply:
IMO that parameter is not required